bibliopixel/drivers/WS2801.py

```diff
@@ -1,18 +1,19 @@
-from . spi_driver_base import DriverSPIBase, ChannelOrder
+from . spi_driver_base import DriverSPIBase, ChannelOrder, SPIInterface
 import os
+from os.path import exists
 from .. import gamma
 
 
 class DriverWS2801(DriverSPIBase):
     """Main driver for WS2801 based LED strips on devices like the Raspberry Pi and BeagleBone"""
 
-    def __init__(self, num, c_order=ChannelOrder.RGB, use_py_spi=True,
-                 dev="/dev/spidev0.0", SPISpeed=1, open=open):
+    def __init__(self, num, c_order=ChannelOrder.RGB, interface=SPIInterface.PY_SPI,
+                 dev="/dev/spidev0.0", SPISpeed=1, open=open, exists=exists):

```

Comment:
I just discovered this:  https://github.com/jmcgeheeiv/pyfakefs- a fake filesystem for unit tests!